### DISCUSSION TITLE
How to download the pcap from security Onion for specific ips..

### DISCUSSION THREAD

**Discussion by tslvr3** [Security-Onion-Solutions/securityonion#11586] [Unsupported Versions] [answered]
When we have drill down the events accordingly we can download the pcap files but if we want to download manually  the pcap file for the specific ips. 

Someone please provide light on this..

Thanks & Regards
Ramana T

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.4/pcap.html:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/a53132cb-c1ef-4b63-99d0-6f469439945c)

Also see https://docs.securityonion.net/en/2.4/stenographer.html#command-line:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/ab922943-cc57-44c2-a08f-a0c32a88ad0a)
